ThreeBotPackages/threefold/tfgrid_directory/jobs/sync_directory.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+from Jumpscale import j
+
+timeformat = "%a, %d %b %Y %H:%M:%S %Z"
+
+def sync_directory():
+    bcdb = j.data.bcdb.get("tf_directory")
+    farm_model = bcdb.model_get(url="tfgrid.farm.1")
+    node_model = bcdb.model_get(url="tfgrid.node.2")
+
+    response = j.clients.threefold_directory.client.ListFarmers()[1]
```

Comment:
probably need to check the response status code before trying to json decoded it.